application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -278,15 +278,23 @@
           #set ($foundUser = false)
           #foreach ($voteObj in $doc.getObjects('XPoll.XPollVoteClass'))
             #set ($user = $voteObj.user)
-            #set ($isCurrentUser = $user == $xcontext.user)
+            #set ($isCurrentUser = $user == $services.model.serialize($services.model.resolveDocument($xcontext.user), 'local'))
```

Comment:
No, this is not right. You need to resolve the voter reference relative to the poll reference and then compare with the current user reference (using equals).